.github/workflows/ci.yml

```
@@ -26,13 +26,24 @@ jobs:
         ./cc-test-reporter before-build
       env:
         CC_TEST_REPORTER_ID: 0caaf0c68cbf5597ac7eb5207944c878a0b3fba46dbfa113093b861f2d25762e
-    - name: Build and test
+    - name: Install and config bundler
       run: |
         gem install bundler:1.17.3
+    - name: Generate 'Gemfile.lock' before caching gems
+      run: bundle lock --update
+    - uses: actions/cache@v2
+      with:
+        path: $HOME/vendor/bundle
```

```suggestion
        path: vendor/bundle
```
I think there's no need to add the $HOME path in here

---

It wanted to use a path not within the project directory to avoid possible override of files.
I'll leave the relative path.
Thanks!